db/create-db.sql

```diff
@@ -0,0 +1,116 @@
+DROP DATABASE IF EXISTS halp;
+CREATE DATABASE halp;
+
+\c halp;
+
+DROP TABLE IF EXISTS employed;
+DROP TABLE IF EXISTS patient;
+DROP TABLE IF EXISTS book;
+DROP TABLE IF EXISTS plate;
+DROP TABLE IF EXISTS menu;
+DROP TABLE IF EXISTS item;
+DROP TABLE IF EXISTS patientOrder;
+DROP TABLE IF EXISTS copyBook;
+DROP TABLE IF EXISTS copyItem;
+
+
+CREATE TYPE employedType AS ENUM ('book', 'kiosk', 'food', 'assistant');
+CREATE TYPE dishType AS ENUM ('starter', 'mainDish', 'dessert');
+CREATE TYPE menuType AS ENUM ('breakfast', 'lunch', 'dinner');
```

Comment:
Finally, we will remove this enum.

Just one attribute for `Menu`: date, with day and time as a primary key.